tests/mocha/field_number_test.js

```diff
@@ -247,6 +247,12 @@ suite('Number Fields', function() {
           numberField.setValue(123.456);
           assertValue(numberField, 123);
         });
+        test('null', function() {
```

Comment:
It's weird to me that you're testing whether the number is set instead of checking what the max/min/precision values on the field are.  Why?

Reply:
Whoops. I was copying the way the previous tests work and didn't think too much about it. Probably makes sense to check the values directly, though that can be a followup since this is a cherry-pick.